### PR TITLE
Add reverse field to fieldOffice in node-health_services_listing

### PR DESCRIPTION
**Pull request by theodur**
## Description
This PR adds the required reverse fields to the `fieldOffice` property inside of the `node-health_services_listing` transformer.

The following changes:
- Unfilter the `reverseFieldRegionPageNode` field in the `node-health_care_region_page`. Previously `reverseFieldRegionPageNode` was filter only reverse fields that were if type `health_care_local_facility`. This was preventing us from retrieving reverse fields of type `regional_health_care_service_des` which is needed for `node-health_services_listing`.

- Refactor the `regional_health_care_service_des` transformer to have it's properties outside of the `entity` property. The properties being inside of the `entity` property was causing errors, because all instances of `regional_health_care_service_des` in the graphQL (`pages.json`) did not have their properties inside of `entity`.

- Only grab needed properties for the `fieldOffice` property. This can be seen [here](https://github.com/department-of-veterans-affairs/vets-website/blob/fe8e20a7a37c72bd38526a7ce0a860725414bb2d/src/site/stages/build/drupal/graphql/healthServicesListingPage.graphql.js#L29).

## Testing done
Built the site locally using the cms export, and visited http://localhost:3001/pittsburgh-health-care/health-services/ to verify the health services were being shown.

## Acceptance criteria
- [ ] The health services listing section ofpittsburgh-health-care/health-services/index.html has no significant diffs when built using cms-export vs. graphql

Other diffs on this page will be covered by additional tickets.

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by mpelzsherman** [comment]
The order of the sections differs from the graphql build, as well as the order of the services within each section.

The sections themselves should be sorted by weight in Drupal, and the services in each section should be sorted alphabetically.

File: src/site/stages/build/process-cms-exports/schemas/output/node-health_services_listing.js [hunk 1]
```diff
@@ -22,7 +25,14 @@ module.exports = {
     fieldMetaTitle: { type: 'string' },
     fieldOffice: {
       type: 'object',
-      properties: { entity: { $ref: 'output/node-health_care_region_page' } },
+      properties: {
+        /* eslint-disable react-hooks/rules-of-hooks */
```
Comment (mpelzsherman) [inline src/site/stages/build/process-cms-exports/schemas/output/node-health_services_listing.js:29]: I guess the react-hooks rule just looks for any function starting with `use`? That's lame. 😆

**Review by mpelzsherman — APPROVED** [review]

**Comment by mpelzsherman** [comment]
It looks like something in this PR broke the side nav - I don't see it in this branch. It's a bit disconcerting that no tests failed as a result! 😮

File: src/site/stages/build/process-cms-exports/schemas/output/node-health_services_listing.js (hunk 1 above)
Comment (mpelzsherman) [reply src/site/stages/build/process-cms-exports/schemas/output/node-health_services_listing.js:29]: Fixed by #14754

File: src/site/stages/build/process-cms-exports/schemas/output/node-health_services_listing.js [hunk 2]
```diff
@@ -22,7 +25,15 @@ module.exports = {
...
```
Comment (U-DON) [inline src/site/stages/build/process-cms-exports/schemas/output/node-health_services_listing.js:29]: You should be good to remove this disable now.

Comment (theodur) [reply src/site/stages/build/process-cms-exports/schemas/output/node-health_services_listing.js:29]: Whoops, forgot about this one.

**Review by Clark-W-Griswold — APPROVED** [review]
👍

**Review by U-DON — APPROVED** [review]

**Pull request merged**